e2e-tests/trade-finance-manager/cypress/integration/journeys/case/deal-page-GEF.spec.js

```diff
@@ -144,6 +151,13 @@ context('User can view a GEF AIN case deal', () => {
   });
 
   describe('facilities table', () => {
+    it('facility coverEndDate should be a dash when date doesn\'t exist', () => {
```

Comment:
need to add "when date doesn\'t exist'" to the last modified test in this file. Can do after this PR